packages/braid-design-system/src/lib/components/Button/Button.tsx

```diff
@@ -370,6 +372,19 @@ export const ButtonText = ({
         ) : null}
         {children}
         {loading ? <ButtonLoader /> : null}
+        {!loading && icon && iconPosition === 'trailing' ? (
```

Comment:
This is what the example looks like in Firefox:

<img width="308" alt="trailing" src="https://user-images.githubusercontent.com/3297808/223586952-b6c72f0a-caa3-4caf-bb5e-a0fe9c97984c.png">

With `white-space: nowrap` on the immediate parent:

<img width="313" alt="nowrap" src="https://user-images.githubusercontent.com/3297808/223587336-3397b3c5-ea62-45be-be24-534944a1f792.png">

Reply:
Yeah found this with Ben earlier, have avoided using `nowrap` deliberately as that has larger implications on the label and its wrapping. Personally I dont think Buttons should wrap, but they do, and we want to avoid a breaking change.